_py1/00-getting-started.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "getting started"
+next_lesson:  data types
+next_link: 01-data-types
+group: "fundamentals"
+num: 0
+video_link: "https://www.youtube.com/embed/eeoSWdtRdbE"
+slides_link: "https://docs.google.com/presentation/d/14gBHUUnuMlXLoHfrBnRulRxCLceZ0Asz9aAaq8rDu0A/edit?usp=sharing"
+length: 14
+agenda: <ul class="py1-lesson-brief-agenda"> 
+            <li> what is python? </li>
+            <li> why is it important? </li> 
+            <li> how is it used? </li> 
+        </ul>
```

Comment:
Hm, I wouldn't encode HTML in your front matter; instead, consider making `agenda` an array, and iterate through each item to render it. Ex:

```yaml
agenda:
  - text: "what is python?"
  - text: "why is it important?"
  - text: "how is it used?"
```

Then, your corresponding liquid code would look something like:

```html
<ul class="py1-lesson-brief-agenda"> 
  {% for item in page.agenda %}
    <li>{{item.text}}</li>
  {% endfor %}
</ul>
```

You can see more about [Lists in YAML](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html) if you're curious!